fcrepo-http-api/src/test/resources/test-objects/test.rdf

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Minor point, but can we shorten this? There doesn't really seem to be any need for all this text for a test resource. Just a few lines of NTriples would do, which you could even just put in a string in the test, no? Unless I'm missing something about how the test works.